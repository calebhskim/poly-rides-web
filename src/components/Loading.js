import React from 'react';

const Loading = () => (
  <div>
    <p>loading...</p>
  </div>
);

export default Loading;
