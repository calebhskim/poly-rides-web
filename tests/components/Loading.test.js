import React from 'react';

import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Loading from '../../src/components/Loading';

describe('<Loading />', () => {
  it('renders properly', () => {
    const wrapper = shallow(<Loading />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
