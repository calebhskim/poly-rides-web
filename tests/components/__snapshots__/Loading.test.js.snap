// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders properly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  <p>
    loading...
  </p>
</div>
`;
